ui/packages/catalog-ui-search/src/main/webapp/exports/wreqr.js

```diff
@@ -0,0 +1 @@
+module.exports = require('wreqr')
```

Comment:
:question: Why does this need to exist? Can this be removed?

Reply:
This is used to expose `wreqr` to downstream distributions. We can guarantee they get the same version as us by exporting this and importing it from DDF there.